### PR TITLE
Handle different input cases for tf.nn.relu6 support

### PR DESCRIPTION
**Pull request by jiafatom**
The [issue](https://github.com/onnx/keras-onnx/issues/505) mentions it needs additional handling for input, which is different from our UT. Add the case here, will ask the issue owner for UT.

### CODE REVIEW

File: keras2onnx/_builtin.py [hunk 1]
```diff
@@ -1726,11 +1726,13 @@ def convert_tf_read_variable_op(scope, operator, container):
...
 def convert_tf_relu6(scope, operator, container):
     oopb = OnnxOperatorBuilder(container, scope)
+    operator_input = operator.raw_operator.input if hasattr(operator.raw_operator, 'input') else \
+        operator.raw_operator.inputs[0]
     oopb.apply_op_with_output("apply_relu6",
                               operator.input_full_names,
                               operator.output_full_names,
                               name=operator.full_name + '_clip',
-                              dtype=operator.raw_operator.input.dtype.as_numpy_dtype)
+                              dtype=operator_input.dtype.as_numpy_dtype)
```
Comment (wenbingl) [inline keras2onnx/_builtin.py:1735]: dtype supposes to be stored in a variable of scope, why is it different here?

Comment (jiafatom) [reply keras2onnx/_builtin.py:1735]: The issue owner mentions there is some case that `operator.raw_operator` does not have attribute 'input', only has 'inputs[0]' instead.

Comment (wenbingl) [reply keras2onnx/_builtin.py:1735]: I see, the previous apply_clip implementation is so bad. can you avoid to call this function?

Comment (jiafatom) [reply keras2onnx/_builtin.py]: synced offline, get the input type from `operator`, not from `raw_operator`

**Pull request merged**